en/mkdocs.yml

```diff
@@ -166,7 +166,9 @@ markdown_extensions:
 # Plugins configuration
 plugins:
   - search
-  - open-in-new-tab
+  - open-in-new-tab:
+      domains:
+        - "bi.docs.wso2.com"
```

Comment:
Is this correct? We should open the external links in new tab. Not the other-way around.